lib/tasks/integration.rake

```diff
@@ -61,7 +61,7 @@ namespace :integrate do
       test_dirs = resolve_test_groups_by_path
 
       {
-        :cucumber_javascript => 'parallel_cucumber --verbose features -o "-b -p parallel --tags=@javascript --tags=~@fakeweb --tags=~@percy --tags=~@ignore"',
+        :cucumber_javascript => 'parallel_cucumber --verbose features -o "-b -p parallel --tags=@javascript --tags=~@percy --tags=~@ignore"',
```

Comment:
All those parallel are not needed anymore

In fact the whole `rake integrate` is not needed anymore